src/portals/hope/features/tools/claim-types/tables/TypesTable/BindingForm.tsx

```diff
@@ -0,0 +1,419 @@
+          ? value
+          : updatingBinding.defaultDeductible,
+    }
+
+    await toast.promise(
+      updateBinding({
+        variables: {
+          id,
+          request,
+        },
+        optimisticResponse: {
+          claimTypeBinding_update: { ...updatingBinding, ...request },
+        },
+        update: (
+          cache: ApolloCache<NormalizedCacheObject>,
+          { data: response },
+        ) => {
+          if (!response?.claimTypeBinding_update) {
+            return
+          }
+
+          const cachedData = cache.readQuery({
+            query: GetClaimTypeBindingsDocument,
+            variables: {},
+          }) as GetClaimTypeBindingsQuery
+
+          const cachedBindings = (cachedData as GetClaimTypeBindingsQuery)
+            ?.claimTypeBindings
+
+          cache.writeQuery({
+            query: GetClaimTypeBindingsDocument,
+            data: {
+              claimTypeBindings: [
+                ...cachedBindings.filter((bind) => bind?.id !== id),
+                { ...updatingBinding, ...request },
+              ],
+            },
+            variables: {},
+          })
+        },
+      }),
+      {
+        loading: `Updating ${convertCamelcaseToTitle(bindingType)}...`,
+        success: `${convertCamelcaseToTitle(bindingType)} updated!`,
+        error: 'Error while updating.',
+      },
+    )
+  }
+
+  const deleteBindingHandler = async (id: string) => {
+    await toast.promise(
+      deleteBinding({
+        variables: {
+          id,
+        },
+        optimisticResponse: {
+          claimTypeBinding_delete: true,
+        },
+        update: (
+          cache: ApolloCache<NormalizedCacheObject>,
+          { data: response },
+        ) => {
+          if (!response?.claimTypeBinding_delete) {
+            return
+          }
+
+          const cachedData = cache.readQuery({
+            query: GetClaimTypeBindingsDocument,
+            variables: {},
+          }) as GetClaimTypeBindingsQuery
+
+          const cachedBindings = (cachedData as GetClaimTypeBindingsQuery)
+            ?.claimTypeBindings
+
+          cache.writeQuery({
+            query: GetClaimTypeBindingsDocument,
+            data: {
+              claimTypeBindings: [
+                ...cachedBindings.filter((bind) => bind?.id !== id),
+              ],
+            },
+            variables: {},
+          })
+        },
+      }),
+      {
+        loading: 'Deleting...',
+        success: 'Deleted!',
+        error: 'Deleting failed.',
+      },
+    )
+  }
+
+  const currency = InsuranceTypeCurrency[insuranceType]
+
+  return (
+    <TypeColumn>
+      <Flex gap="tiny">
+        <Checkbox
+          checked={!!binding}
+          onChange={() =>
+            binding
+              ? deleteBindingHandler(binding.id)
+              : createBindingHandler(claimType, insuranceType)
+          }
+        />
+        <BindingContent>
+          <span>{binding ? 'Binded' : 'Enable Binding'}</span>
```

Comment:
I think "Bounded" is the correct word, but I think "enabled" is even better here
```suggestion
          <span>{binding ? 'Enabled' : 'Enable Binding'}</span>
```